Nos/Service/CurrentUser.swift

```diff
@@ -154,11 +154,8 @@ enum CurrentUserError: Error {
         
         // Recommended Relays for new user
         for address in Relay.recommended {
-            _ = try? Relay(
-                context: viewContext,
-                address: address,
-                author: author
-            )
+            let relay = try? Relay.findOrCreate(by: address, context: viewContext)
```

Comment:
Changing this so that it doesn't assign the relay to the author may have fixed #1213.